### ISSUE TITLE
Suggestion - Custom definition

### ISSUE THREAD

**Issue by generrosity** [GenDeathrow/MoreChickens#50]
Apologies - I tried google and searching here.

Is it possible to define custom chickens?

I assume not, as it would need a texture file to be inserted somewhere. But it would be quite a interesting idea.

Context - our little server was been nicknamed 'waffleton' due to the amount of useless items that come up sometimes. Some of our younger players use More Chickens to help them with getting through mods they dont yet understand. I was thinking of building a little tree of chickens to tie the different mods together and be unique, like harvestcraft:salt + botania:manasteel = harvestcraft:waffle, waffle + RS:quartz_enriched = <specific minor OP related item> etc.

**Comment by ExZha1337** [comment]
This would be super cool

**Comment by GenDeathrow** [comment]
not sure why I hadn't seen this post, but apparently it got lost. It is not possible to define custom chickens. although Im really been thinking of adding this capability recently.

**Comment by ExZha1337** [comment]
Yeah, it would be super cool. Since i tried adding the new alloys from "Ender IO Endergy" mod in to the chickens through the config. The only problem i can see would be skins for the chickens, but you probably already know a way around that problem.

**Comment by Teibidh** [comment]
Could just be a generic texture with a user defined color scheme or something... set the color in the config file when you add the material. The resulting chicken is vanilla chicken white with a candy cane stripe of whatever the set color was.

This would be cool, and would allow others to publish their "chicken configs" for people to use instead of you having to go manually add chickens when something new comes up.
